### DISCUSSION TITLE
Allow to add mods to the game the same way ruleset are added (DLL)

### DISCUSSION THREAD

**Discussion by alexpado** [ppy/osu#24376] [Ideas] [answered]
Hi,

It would be cool if it was possible to load custom mods through a DLL file just like it is done with rulesets.

One major issue I do see with this is "How to declare to which ruleset your mod applies ? as many rulesets can be added"... And I don't have any answers to that.

This would allow to send the mod to players for them to try it without having to clone the repo, install dependencies, build the project etc...

**Comment by peppy** [comment, marked as answer]
Sorry but this is not happening.
